src/projections/ortho.cpp

```diff
@@ -121,24 +131,180 @@ static PJ_LP ortho_s_inverse (PJ_XY xy, PJ *P) {           /* Spheroidal, invers
 }
 
 
+static PJ_XY ortho_e_forward (PJ_LP lp, PJ *P) {           /* Ellipsoidal, forward */
+    PJ_XY xy;
+    struct pj_opaque *Q = static_cast<struct pj_opaque*>(P->opaque);
+
+    // From EPSG guidance note 7.2
```

Comment:
Do you have a more specific reference to this? Equations x.y.z or chapter a.b.c?

Reply:
done